### DISCUSSION TITLE
Feed "IN_PROGRESS" for 10 days

### DISCUSSION THREAD

**Discussion by akg397** [amzn/selling-partner-api-models#3638] [General]
I have a feed that been stuck "IN_PROGRESS" for 10 days.
I have ticket 13086764751 to support open for 9 days.
There is just no action on anything. What should I do. I would love to have the issue escalated to somebody who can look at amazon's backend system. Any ideas how to escalate ?

**Comment by vaidas-lungis** [comment]
Sorry I can't help you with the questions, but have You thought about cancelling the feed (https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2021-06-30-reference#cancelfeed) and resubmit the data?

**Reply by akg397** [reply]
" Only feeds with processingStatus=IN_QUEUE can be cancelled"

So once its stuck in "IN_PROGRESS" - there is no way out
